types/src/proto/validator_change.proto

```diff
@@ -5,19 +5,14 @@ syntax = "proto3";
 
 package types;
 
-import "events.proto";
 import "ledger_info.proto";
 
-// This is used to prove validator changes.  When a validator is changing, it
-// triggers an event on /validator_change_account/events/sent.  To tell the
-// client about validator changes, we query
-// /validator_change_account/events/sent to get all versions that contain
-// validator changes after the version that we are trying to update from. For
-// each of these versions, the old validator set would have signed the ledger
-// info at that version.  The client needs this as well as the event results +
-// proof.  The client can then verify that these events were under the current
-// tree and that the changes were signed by the old validators (and that the
-// events correctly show which validators are the new validators).
+// This is used to prove validator changes.
```

Comment:
you like simplicity lol

Reply:
The entire old comment block was out-dated :( Maybe I can write something new.